sections/semantics-forms.include

```diff
@@ -3761,6 +3761,8 @@
   When the element is <a>suffering from a step mismatch</a>, the user agent may round the
   element's <a for="forms">value</a> to the nearest <a for="dates">date</a> for which the element would not <a>suffer from a step mismatch</a>.
 
+  The <code>change</code> event is fired whenever the user has selected a new <a>valid date string</a> - which can be triggered when the user enters a new value. When a user changes a new <a>valid date string</a> or uses a stepping function to increment or decrement a part of the overall date string, user agents may apply a timeout to determine that the changed value is no longer intermediate. At the completion of such timeout, <code>change</code> must fire. Moving focus away from the date picker component or explicitly confirming with a button are unambigous triggers that the user wants the value changed, and <code>change</code> should fire on all such cases if the new date value is different than the previous one. 
+
```

Comment:
Nitpicking: Can you please break up the line to around 100 characters? At the end of HTML elements is often a good spot.

I would move the "when the user changes, or is stepping though" clause to the end.

In that piece I don't think it  should say that user agents must fire anything, just make it clear that it is possible they do so.